packages/module-federation/src/executors/utils/start-remote-iterators.ts

```diff
@@ -0,0 +1,123 @@
+import { StartRemoteFn, type StartRemoteIteratorsOptions } from './models';
+import {
+  getModuleFederationConfig,
+  getRemotes,
+  parseStaticRemotesConfig,
+  parseStaticSsrRemotesConfig,
+  startRemoteProxies,
+  startSsrRemoteProxies,
+} from '../../utils';
+import { buildStaticRemotes } from './build-static-remotes';
+import {
+  startSsrStaticRemotesFileServer,
+  startStaticRemotesFileServer,
+} from './start-static-remotes-file-server';
+import {
+  type ExecutorContext,
+  readProjectsConfigurationFromProjectGraph,
+} from '@nx/devkit';
+
+export async function startRemoteIterators(
```

Comment:
nit: What about just `startRemotes`, `serveRemotes` or `runRemotes`? That's what the function does and the iterators are just an implementation detail.

Reply:
It is an impl detail, but I think it makes sense in this case as it indicates that you're getting iterators for both static and dev remotes, which are then combined thereafter.

If we wanted to rename, `runRemotes` might be the best option, but that can be a cleanup for a different time if we decide on it.